### ISSUE TITLE
No packages for ubuntu/trusty

### ISSUE THREAD

**Issue by pfak** [sensu/sensu-go#2520]
There is presently no package built for `sensu-go-{agent,cli,backend}` for ubuntu/trusty (14.04).

## Expected Behavior

Package to be built for x86_64 ubuntu/trusty and deployed to https://packagecloud.io/sensu/stable.

## Current Behavior

No packages available to be installed on ubuntu/trusty.

## Possible Solution

I was able to manually build a package using the docker image for ubuntu/trusty, but it would be nice to have an official one.

## Steps to Reproduce (for bugs)
```# curl -s https://packagecloud.io/install/repositories/sensu/stable/script.deb.sh | sudo bash
...
# apt-get install sensu-go-agent
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package sensu-go-agent
```

## Context

Trying to monitor Ubuntu 14.04 hosts.

## Your Environment

sensu-go 5.0 on Ubuntu 14.04 (trusty).

**Comment by amdprophet** [comment]
Hi @pfak,

Thank you for opening this issue! We are planning on adding support for ubuntu/trusty for the upcoming 5.0.2 release slated for next week.

**Comment by echlebek** [comment]
The 5.0.2 release is becoming the 5.1.0 release, so I've updated the milestone.

As our packaging lives in a private repository here, there is no PR to associate with this issue. However, we have added the distribution and it will be available with the 5.1.0 release. Thank you for filing this issue. :)

**Comment by R725853** [comment]
Hi, I am running the  same issue here. Trying do install sensu-go on Ubuntu 20.04 LTS and after runnig:
curl -s https://packagecloud.io/install/repositories/sensu/stable/script.deb.sh | sudo bash
sudo apt-get install sensu-go-backend
i get the output:
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package sensu-go-backend
Is this the right repository... Can you tell me what´s wrong?
